### ISSUE TITLE
Changes to install on Laravel ^6.2

### ISSUE THREAD

**Issue by TiagoSilvaPereira** [lukepolo/laracart#261]
Hi,

To install on Laravel ^6.2, I needed to change:

```
"lukepolo/laracart": "1.4.*"
```
to
```
"lukepolo/laracart": "^1.4"
```

Hope this helps someone.

thanks

**Comment by lukepolo** [comment]
Thanks! Does 1.5 not work for you?

**Comment by TiagoSilvaPereira** [comment]
Hi @lukepolo thanks for responding.

The version 1.5 installed well here :smile: 

But I'm having a little trouble here... I have an item with two sub-itens, the first with quantity = 2 and the second with quantity = 0.

But when I call

```php
$item->subTotal();
```
It sums the price of the sub-item with quantity 0 too.

Here is the structure of my item:

![image](https://user-images.githubusercontent.com/11933789/73483799-d6fbfc00-437e-11ea-929f-8d0818eddedc.png)

It is returning 372.00 in the price, but the correct is 304.00.

Am I doing something wrong?

Thanks!!

**Comment by lukepolo** [comment]
There is a bug on

https://github.com/lukepolo/laracart/blob/master/src/CartItem.php#L265

It will always return 1 even if its 0.

**Comment by TiagoSilvaPereira** [comment]
thanks, for now I'll avoid to add items with quantity 0

**Comment by lukepolo** [comment]
Ill change the code and see if the tests pass. Then ill release a new version here in a few.

**Comment by lukepolo** [comment]
Fixed with : 1.8.4

**Comment by TiagoSilvaPereira** [comment]
thanks @lukepolo you rocks bro!!!
